src/CardList/CardList.js

```diff
@@ -3,29 +3,24 @@ import CardListItem from '../CardListItem/CardListItem.js';
 import './CardList.css'
 
 class CardList extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
```

Comment:
There were a lot of warning messages that read 'useless constructor'. I noticed that if the constructor is not doing anything other than calling `super()`, the method can be deleted. This gets rid of the warning message.

Reply:
This could probably be made into a functional component then.